snow/consensus/snowball/tree_test.go

```diff
@@ -176,26 +172,21 @@ func TestSnowballAddPreviouslyRejected(t *testing.T) {
 	zero := ids.ID{0b00000000}
 	one := ids.ID{0b00000001}
 	two := ids.ID{0b00000010}
-	four := ids.ID{0b00000100}
 
 	params := Parameters{
 		K:               1,
 		AlphaPreference: 1,
 		AlphaConfidence: 1,
-		BetaVirtuous:    1,
-		BetaRogue:       2,
+		Beta:            2,
```

Comment:
Re-write this test based off the name. The previous behavior relied on divergent beta values for virtuous/rogue instances.